react/features/base/i18n/i18next.js

```diff
@@ -81,4 +82,9 @@ i18next.addResourceBundle(
     /* deep */ true,
     /* overwrite */ true);
 
+// Add builtin languages, if applicable.
+if (BuiltinLanguages && BuiltinLanguages.setup) {
+    BuiltinLanguages.setup();
+}
+
```

Comment:
Is order really important here? Because if order isn't really necessary, we might as well simplify this by not exporting `setup` from `BuiltinLanguages` and doing the whole initialization in BuiltinLanguages.native.js.

Reply:
I'll explore this idea!